src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -136,8 +141,39 @@ const useStyles = makeStyles({
       marginTop: '-3px',
     },
   },
-  editButtonCell: {
-    padding: '0',
+  editContainerInTable: {
+    borderTop: `1px solid ${theme.altinnPalette.primary.blueLight}`,
+    marginBottom: 0,
+  },
+  editContainerRow: {
+    '&:hover': {
+      background: 'unset !important',
+    },
+  },
+  editingRow: {
+    backgroundColor: 'rgba(227, 247, 255, 0.5)',
+    '& td': {
+      borderBottom: 0,
+      '&:nth-child(1)': {
```

Comment:
I just glanced at this, but could we have a more specific class here? :thinking: It's hard for me to understand what 'the second child of every `td`' means while reading this, at least. I also noticed below that you add some padding for every `span` inside here - that seems dangerous if this container can include anything.

Reply:
This whole block of css is in order to get a dotted top-border for the row being edited. In tables, the `dotted` border-style behaves in a way that makes it uneven when adding it to a row consisting of several table-cells. 

A solution i found was to create a psuedo-element to the first cell of the current row being edited and add the border to that instead. In order to align the border correctly i had to remove the padding from the cell itself. Then the content would be placed all the way to the left, so i added a span inside each cell around its content and added the same padding that was removed from the cell itself to the span of the first cell in the row currently being edited.

I could definitely try to create a specific class for adding the padding and add it to the `containter-span` for the cells.

Any other improvements to this fix or alternative solutions are very welcome 😄

Reply:
Ahhh, I see! :exploding_head: I inspected this in devtools now, and I absolutely get it.. Clever fix, and I'm happy to sign off on it! Adding code to inject a class here (and not just do it in CSS) might just make it harder to maintain, so I vote for just keeping it as-is.